awesome/database/migrations/20230820065806_020_insert_directus_fields_UserCourses_courseId.js

```diff
@@ -0,0 +1,36 @@
+/**

```

Comment:
All such code is duplicated. Could you refer to this? https://github.com/ocademy-ai/machine-learning/blob/main/awesome/database/migrations/20230815040913_012_insert_directus_fields_constrain_for_ids.js

You can create a helper function in utils.js as `addDirectusFKInterface(collection, field)`

Reply:
done